admin-dev/themes/new-theme/js/pages/address/form.js

```diff
@@ -40,11 +40,15 @@ $(document).ready(() => {
       company: addressFormMap.addressCompanyInput,
     },
   );
+
+  const addressStateBlock = $(addressFormMap.addressStateBlock).closest('.form-group.row');
```

Comment:
Not a good idea. If someone changes the parent, it will break the js.
Change the addressStateBlock selector instead.

Reply:
Fixed by adding class to the whole row via 
```
 'row_attr' => [
      'class' => 'js-address-state-block',
],
```

And using that class as selector.